src/EFCore.PG.NodaTime/Storage/Internal/NpgsqlNodaTimeTypeMappingSourcePlugin.cs

```diff
@@ -137,41 +137,55 @@ protected virtual RelationalTypeMapping FindExistingMapping(in RelationalTypeMap
                 : mapping;
         }
 
-        RelationalTypeMapping FindArrayMapping(in RelationalTypeMappingInfo mappingInfo)
+        // TODO: This is duplicated from NpgsqlTypeMappingSource
+        protected virtual RelationalTypeMapping FindArrayMapping(in RelationalTypeMappingInfo mappingInfo)
         {
-            // PostgreSQL array type names are the element plus []
+            var clrType = mappingInfo.ClrType;
```

Comment:
When could `clrType` be equal to `null`?

Reply:
When reverse engineering an existing database - all you have is the store type from PostgreSQL, and you're creating the C# code model.